apps/tunablelaser/values-summit.yaml

```diff
@@ -0,0 +1,11 @@
+namespace: calsys
+image:
+  repository: ts-dockerhub.lsst.org/tunablelaser
+  tag: c0036
+  pullPolicy: Always
+  nexus3: nexus3-docker
+env:
```

Comment:
You also need to specify  `OSPL_INFOFILE` and `OSPL_ERRORFILE`. The template usually is:
`OSPL_INFOFILE: /tmp/ospl-info-<csc>.log`
`OSPL_ERRORFILE: /tmp/ospl-error-<csc>.log`